crypto/ocsp/ocsp_vfy.c

```diff
@@ -419,8 +420,8 @@ int OCSP_request_verify(OCSP_REQUEST *req, STACK_OF(X509) *certs,
 }
 
 static int ocsp_req_find_signer(X509 **psigner, OCSP_REQUEST *req,
-                                const X509_NAME *nm, STACK_OF(X509) *certs,
-                                unsigned long flags)
+                                const X509_NAME *nm, const
+                                STACK_OF(X509) *certs, unsigned long flags)
```

Comment:
The added `const` for `STACK_OF(X509) *certs` should be on the same line.

Reply:
Thanks for your swift approval!

> The added const for STACK_OF(X509) *certs should be on the same line.

Changed.

Though there are already several such occurrences in our codebase:
```
find . -name "*.[ch]" | xargs grep -e "const\$" | grep -v "away the const"

./crypto/cmp/cmp_hdr.c:ASN1_OCTET_STRING *OSSL_CMP_HDR_get0_transactionID(const
./crypto/ec/ecp_nistputil.c:                                                                      const
./crypto/x509/x509cset.c:const STACK_OF(X509_EXTENSION) *X509_REVOKED_get0_extensions(const
./crypto/x509/pcy_lib.c:STACK_OF(X509_POLICY_NODE) *X509_policy_tree_get0_policies(const
./crypto/x509/pcy_lib.c:STACK_OF(X509_POLICY_NODE) *X509_policy_tree_get0_user_policies(const
./crypto/x509/pcy_lib.c:STACK_OF(POLICYQUALINFO) *X509_policy_node_get0_qualifiers(const
./crypto/ess/ess_lib.c:                                                       const
./crypto/whrlpool/wp_block.c:static const
./include/openssl/x509.h:                                                        const
./include/openssl/x509.h:                                                        const
./include/openssl/cmp.h:ASN1_OCTET_STRING *OSSL_CMP_HDR_get0_transactionID(const
./include/openssl/safestack.h: * In practice, however, it appears to be hard to have a const
./include/openssl/ess.h:                                                       const
```